src/index.html

```diff
@@ -5,9 +5,251 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="icon" href="images/favicon.png">
   </head>
+
   <body>
-    <h1>Miami</h1>
+
+    <header class="header">
+      <img
+      src="images/Logo_Miami.svg"
+      class="header__logo-image"
+      alt="miami-logo"
+      >
+      <a href="#" class="header__phone-wrapper">
+        <div></div>
+        <img
+          src="images/Icon-Phone-call.svg"
+          class="header__phone-icon"
+          alt="phone-icon"
+        >
+      </a>
+      <a href="#">
+        <img
+          src="images/Icon-Burger-menu.svg"
+          class="header__burger-icon"
+          alt="burger-icon"
+        >
+      </a>
+      <h1 class="title header__title">
+        <span></span>
+        Miami <br> Condo Kings
+      </h1>
+    </header>
+
+    <main class="main">
+      <section class="intro main__intro">
+        <div class="intro__wrapper">
+          <h2 class="heading intro__heading">
+            Miami Best <br> Real estate
+            <span></span>
+          </h2>
+          <article class="description-text intro__description-text">
+            <p>
+              Over 50 years experience in love.
+            </p>
+            <p>
+              Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.
+            </p>
+            <p>
+              The market leader in real estate since 2003. Awards Real Estate Company, 2010, 2011, 2013. More than 200 satisfied customers premium segment. We focus only on new buildings and high-level doskanalno know this area better than other specialists.Working directly with developers - so have more useful information about the profitable deals.
+            </p>
+            <p>
+              We will never be late!<br>Guaranteed secure transaction - 100%
```

Comment:
```suggestion
              We will never be late!
              <br>
              Guaranteed secure transaction - 100%
```
For better readability it's better to move `<br>` to the newline